spread-tests/hello-world-is-confined/task.yaml

```diff
@@ -0,0 +1,13 @@
+summary: Check that hello-world.evil is confined
+# This is blacklisted on distributions without full confinement
+systems: [-debian-sid-grub]
+restore: |
+    snap remove hello-world
+execute: |
+    cd /
+    echo Run some hello-world stuff
+    snap install hello-world
+    hello-world.echo | grep Hello
```

Comment:
This test may be ported to use test-snapd-tools, which offers saner apps for this sort of testing. Rather than "evil", try to exercise the system in some way that is expected to break with "cat /foo", for example.